addons/mail/static/src/discuss/core/common/notification_settings.xml

```diff
@@ -4,6 +4,11 @@
     <t t-name="discuss.NotificationSettings">
         <ActionPanel title.translate="Notification Settings">
             <div class="o-discuss-NotificationSettings">
+
+                <div class="d-flex flex-column text-truncate alert alert-warning">
+                    <span><i class="fa fa-exclamation-triangle"/> Server is muted</span>
+                </div>
+
```

Comment:
I assume this should be only visible when `store.settings.mute_until_dt`.

I would also add something to quickly open notification settings there, so that you can easily unmute server in case you forgot where to unmute. Especially when this is located close-by

Reply:
Also no `text-truncate`, this is overrated and not good UX/UI. Let it wrap 🌮

Reply:
ho my bad, this part still need some work, I forgot about it